lib/utils.js

```diff
@@ -307,8 +307,16 @@ function prettifyObject ({
 
   let result = ''
 
-  const keysToIterate = objectKeys.filter(k => keysToIgnore.includes(k) === false)
-  for (var i = 0; i < objectKeys.length; i += 1) {
+  function safeObjectKeys () {
```

Comment:
Could we move this declaration below `return result`? It'll be hoisted appropriately and I think the organization is easier to read.

Reply:
Sure, that makes sense. I've updated the method placement.